packages/vanilla/src/components/global-nav/top-nav/search/search.html

```diff
@@ -1,18 +1,19 @@
-<div class="hig__global-nav__top-nav__search hig__global-nav__top-nav__item">
-  <div class="hig__global-nav__top-nav__search__content">
-    <label for="top-nav-search" class="hig__global-nav__top-nav__search__icon"></label>
-    <input
-      name="top-nav-search"
-      id="top-nav-search"
-      class="hig__global-nav__top-nav__search__input"
-      type="text"
-      placeholder="{{placeholder}}"
-      value="{{query}}"
-    >
-    <div class='hig__global-nav__top-nav__search__clear'></div>
-  </div>  
-  <div class="hig__option__wrapper">
-    <div class="hig__option__list"></div>
+<div class="hig__global-nav__top-nav__item">
```

Comment:
Again, moved `.hig__global-nav__top-nav__item` one level higher to prevent rule conflicts.